taikoxyz/taiko-mono
# chore(repo): add taiko-client at 8159155

> [!NOTE]  
> This will fail until the other PR https://github.com/taikoxyz/taiko-mono/pull/16340 is merged in.

## summary
- adds taiko-client at `8159155` with no other changes, except for `rm -rf .git`.

## how to review
1. checkout the branch: `git checkout add-client-only`
2. clone the current taiko-client: `git clone git@github.com:taikoxyz/taiko-client.git taiko-client-old`
3. make sure they are the exact same: `diff -r taiko-client taiko-client-old`

**expected output**:
<img width="355" alt="image" src="https://github.com/taikoxyz/taiko-mono/assets/13951458/ba16dac6-3206-4466-bd60-6b5793f0820c">
